sklearn/cross_decomposition/__init__.py

```diff
@@ -1,2 +1,2 @@
-from .pls_ import *
-from .cca_ import *
```

Comment:
The right fix would have been to get rid of this "import *" (which is horrible), and only import the estimators, then exposed in __all__.

Hopefully in another PR?